tools/PI/DevHome.PI/Pages/WERPage.xaml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Page
+    x:Class="DevHome.PI.Pages.WERPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
+    xmlns:models="using:DevHome.PI.Models"
+    mc:Ignorable="d"
+    NavigationCacheMode="Enabled">
+
+    <Grid>
+        <Grid.Resources>
+            <Style TargetType="TextBlock">
+                <Setter Property="Margin" Value="0,0,0,6"/>
+            </Style>
+        </Grid.Resources>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="2*"/>
+            <RowDefinition Height="3*"/>
+        </Grid.RowDefinitions>
+
+        <StackPanel Orientation="Vertical">
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <TextBlock x:Uid="WERHeaderTextBlock" FontSize="{StaticResource SubtitleTextBlockFontSize}" FontWeight="SemiBold" Margin="0,0,0,8" Grid.ColumnSpan="2"/>
+
+                <StackPanel x:Name="AppLocalDumpCollection" Visibility="{x:Bind ViewModel.AttachedToApp, Mode=OneWay}" Orientation="Horizontal" Spacing="5" Grid.Row="1" Grid.Column="1">
+                    <TextBlock x:Uid="LocalDumpCollectionLabel" VerticalAlignment="Center" Margin="0 0 10 0"/>
+                    <ToggleSwitch x:Uid="LocalDumpCollectionToggle" x:Name="LocalDumpCollectionToggle" IsOn="{x:Bind ViewModel.LocalCollectionEnabledForApp, Mode=TwoWay}" IsEnabled="{x:Bind ViewModel.RunningAsAdmin}" Toggled="LocalDumpCollection_Toggled" Margin="0 0 -70 0" />
+                    <Button Visibility="{x:Bind ViewModel.AllowElevationOption}" Command="{x:Bind ViewModel.RunAsAdminCommand}">
+                        <StackPanel Orientation="Horizontal" Spacing="10">
+                            <SymbolIcon Symbol="Admin"/>
+                            <TextBlock x:Uid="RunElevatedButtonForWER" Margin="0"/>
+                        </StackPanel>
+                    </Button>
+                    <HyperlinkButton x:Uid="LocalDumpMoreInfo" NavigateUri="https://learn.microsoft.com/windows/win32/wer/collecting-user-mode-dumps" />
```

Comment:
Probably should use a forwarding link here.

Reply:
Ok